drivers/gles2/shaders/scene.glsl

```diff
@@ -1466,9 +1466,6 @@ void main() {
 	float anisotropy = 0.0;
 	vec2 anisotropy_flow = vec2(1.0, 0.0);
 	float sss_strength = 0.0; //unused
-	// gl_FragDepth is not available in GLES2, so writing to DEPTH is not converted to gl_FragDepth by Godot compiler resulting in a
-	// compile error because DEPTH is not a variable.
-	float m_DEPTH = 0.0;
```

Comment:
You shouldn't mix another change into a PR. Always stick to one topic, if you want to make other changes you should make multiple PRs.

In this case, removing ``m_DEPTH`` will cause a shader compile error when users try to access ``DEPTH`` in the fragment shader.

Reply:
It was my mistake. I fixed and also squashed the commits.